lab/spending_guidelines.md

```diff
@@ -0,0 +1,83 @@
+# OPI Lab Budget Guidelines
+
+This document provides guidance for using OPI funds to acquire lab
+equipment.  We describe both "appropriate" and "inappropriate" use of
+equipment funds and include examples of each.  Generally, commonly
+used lab equipment generally available in the market are appropriate.
+Pre-production equipment or equipment that do not service the
+goals of the OPI lab are inappropriate.
+
+
+## Appropriate
+
+### General purpose servers
+
+- 1U, 2U, 3U, 4U rack servers with PCIE 3.0/4.0/5.0 slots
+- Pedestal servers
+
+### Server peripherals
+
+- Hard drives
+- RAM
+- NIC cards
+
+### Test equipment
+
+- IXIA traffic generators
```

Comment:
Should this be genericized to simply "traffic generators"? i.e. not specifying one brand/product line of traffic generators?

Reply:
Sounds reasonable to me.